pyproject.toml

```
@@ -18,3 +18,18 @@ showcontent = true
 directory = "bugfix"
 name = "Bug fixes:"
 showcontent = true
+
+[tool.poetry]
+name = "plonetheme.barceloneta"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

```suggestion
authors = ["The Plone Community and its Release Managers <releasemanager@plone.org>"]
```
(no idea if there's something better)